libr/debug/p/native/linux/linux_debug.c

```diff
@@ -195,10 +214,27 @@ bool linux_set_options (RDebug *dbg, int pid) {
 
 int linux_attach (RDebug *dbg, int pid) {
 	linux_set_options (dbg, pid);
-	int ret = ptrace (PTRACE_ATTACH, pid, 0, 0);
+	int ret = ptrace (PTRACE_ATTACH, pid, NULL, NULL);
 	if (ret != -1) {
 		perror ("ptrace (PT_ATTACH)");
 	}
+	
+	if (dbg->h) {
+		RList *list = dbg->h->threads (dbg, pid);
```

Comment:
don't use dbg->h->threads since you are already in the linux backend, instead use linux_thread_list